.gitignore

```diff
@@ -19,8 +19,7 @@ target
 /esvm
 .htpasswd
 .eslintcache
-plugins
-data
+installedPlugins
```

Comment:
`usr1@elk-vm1:~/dev/kibana-i18n$ git diff master -- .gitignore
diff --git a/.gitignore b/.gitignore
index 4f69764..74345c7 100644
--- a/.gitignore
+++ b/.gitignore
@@ -28,3 +28,4 @@ selenium
 .babelcache.json
 *.swp
 *.swo
+data/translations
`
It may have been changes for a previous commit. The above diff with master doesn't show those changes.